docs/release-notes/gift-card-support-multi-currencies.md

```diff
@@ -0,0 +1,29 @@
+---
+slug: "gift-card-support-multi-currencies"
+title: "Gift card support multi-currencies"
+createdAt: 2023-07-27T12:00:00.000Z
+hidden: false
+type: "added"
+excerpt: "Configure your store gift cards in multiple currencies."
+---
+As of July 30, 2023, you will be able to create gift cards in currencies other than those added to your store. This new feature allows you to indicate in which currency the gift card should be created when adding a new gift card in the VTEX Admin.
+ 
+Also, a new optional field (`currencyCode`) has been added to the endpoints below, enabling you to identify the currency applicable to each gift card.
+
+**GiftCard API**
```

Comment:
Por que a API é `GiftCard`e quando se fala da funcionalidade é `gift card`?

Reply:
A Clara Cluz (localization) explicou que quando for nome de API, é para manter junto (GiftCard).
Thread: https://vtex.slack.com/archives/C052CEU8GDU/p1689276373065159?thread_ts=1689273609.240369&cid=C052CEU8GDU